xcm/xcm-simulator/example/src/lib.rs

```diff
@@ -24,6 +24,7 @@ use xcm_executor::traits::ConvertLocation;
 use xcm_simulator::{decl_test_network, decl_test_parachain, decl_test_relay_chain, TestExt};
 
 pub const ALICE: sp_runtime::AccountId32 = sp_runtime::AccountId32::new([0u8; 32]);
+pub const BOB: sp_runtime::AccountId32 = sp_runtime::AccountId32::new([0u8; 32]);
```

Comment:
shouldn't it be:
```suggestion
pub const BOB: sp_runtime::AccountId32 = sp_runtime::AccountId32::new([1u8; 32]);
```
?

Reply:
You are completely right, and this will probably screw my test. Thanks!